### DISCUSSION TITLE
Trouble downloading lua, curl, others. 404 from ghcr.io

### DISCUSSION THREAD

**Discussion by gtoast** [Homebrew/discussions#1812] [Getting started] [answered]
Getting the errors when trying to install formulas. Seems to be realted to ghrc.io somehow. Any help is much appreciated!

```
gtoast ~ % brew install curl
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 1 formula.

...

==> Downloading https://ghcr.io/v2/homebrew/core/openldap/manifests/2.4.57
##O=#  #
curl: (22) The requested URL returned error: 404
Error: Failed to download resource "openldap_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/openldap/manifests/2.4.57
```

```
gtoast ~ % brew upgrade lua
==> Upgrading 1 outdated package:
lua 5.3.5_1 -> 5.4.2
==> Upgrading lua
  5.3.5_1 -> 5.4.2

==> Downloading https://ghcr.io/v2/homebrew/core/lua/manifests/5.4.2
##O#- #
curl: (22) The requested URL returned error: 404
Error: Failed to download resource "lua_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/lua/manifests/5.4.2
```

**Comment by SMillerDev** [comment]
What does `brew config` and `brew doctor` say?

**Comment by carlocab** [comment, marked as answer]
The latest versions of `lua` and `openldap` should be `5.4.3` and `2.5.5` respectively, so there's something wrong with your local core repository. Try
```
brew update
git -C $(brew --repo homebrew/core) checkout master
```
If that doesn't work, please show the output of `brew config` and `brew doctor` as requested.

**Reply by gtoast** [reply]
I think this cleared things up. When I ran the git command it balked at a file that needed to be committed. After blowing that file away, I updated  brew and a ton more recipes were  reported as outdated.

I was experimenting with writing my own formula a while ago. A bad formula must have ended up in the repo somehow and was preventing my homebrew repository from getting updated. 

Thanks so much!
